src/python/bot/tasks/minimize_task.py

```diff
@@ -1122,25 +1145,15 @@ def _unset_dedup_flags():
         testcase_file_path, file_host.rebase_to_worker_root(testcase_file_path))
     rebased_output_file_path = file_host.rebase_to_worker_root(output_file_path)
 
-  arguments = environment.get_value('APP_ARGS', '')
-  arguments += (' --cf-{tool_name}-timeout={timeout} '
-                '--cf-{tool_name}-to={output_file_path}').format(
-                    tool_name=tool_name,
-                    output_file_path=rebased_output_file_path,
-                    timeout=timeout)
-  command = testcase_manager.get_command_line_for_application(
-      file_to_run=testcase_file_path,
-      app_args=arguments,
-      needs_http=testcase.http_flag)
-  logs.log('Executing command: %s' % command)
+  arguments = data_handler.get_arguments(testcase, strip_target=False).split()
+  target_name = arguments.pop(0)
 
   if set_dedup_flags:
     _set_dedup_flags()
 
-  # A small buffer is added to the timeout to allow the current test to
-  # finish, and file to be written. Since we should terminate beforehand, a
-  # long delay only slows fuzzing in cases where it's necessary.
-  _, _, output = process_handler.run_process(command, timeout=timeout + 60)
+  result = run_libfuzzer_engine(tool_name, target_name, arguments,
+                                testcase_file_path, rebased_output_file_path,
+                                timeout)
```

Comment:
Does the 60 sec buffer need to be added to timeout ?

Reply:
I don't think this is necessary anymore, since we don't having a timeout which needs to encompass both the launcher process + the actual libFuzzer minimize.